package.json

```diff
@@ -28,6 +28,7 @@
   "homepage": "https://github.com/codecov/uploader#readme",
   "dependencies": {
     "glob": "7.1.7",
+    "js-yaml": "4.1.0",
```

Comment:
If we are parsing the codecov.yml, we should consider the `ignore` section when we implement the correct igmore functionality. If the customer doesn't want certain paths reported on, those paths probably don't contain coverage files.